massa-execution/src/config.rs

```diff
@@ -10,4 +11,12 @@ pub const CHANNEL_SIZE: usize = 256;
 pub struct ExecutionSettings {
     /// Initial SCE ledger file
     pub initial_sce_ledger_path: PathBuf,
+    /// Thread count
+    pub thread_count: u8,
```

Comment:
does this mean that the `execution` section of the config file needs to contain a duplicate of parameters like thread_count ?

Reply:
how is clock_compensation defined ? it should be dynamically computed at every bootstrap and passed

Reply:
solved